kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/configuration/ConfigurationService.java

```diff
@@ -80,6 +80,13 @@
 	 */
 	public ComponentConfiguration getComponentConfiguration(String pid) throws KuraException;
 	
+	/**
+	 * Returns component configuration properties for specified PID
+	 * @param pid - the ID of the component whose configuration is requested
+	 * @return component configuration properties
+	 * @throws KuraException
+	 */
+	public Map<String, Object> getConfigurableComponentConfigurationProperties(String pid) throws KuraException;
```

Comment:
@ibinshtok I know that @LucaDazi is working on similar methods in his multi ton branch. Maybe we should get in touch with him in order to know if there is a similar method in his implementation that can be used here